src/main/java/com/parallax/server/blocklyprop/db/dao/UserDao.java

```diff
@@ -16,10 +16,29 @@
  */
 public interface UserDao {
 
+    @Deprecated
```

Comment:
Does this do something actively?

Reply:
The @Deprecated directive will force the build system to generate a warning when it encounters code that attempts to use the deprecated function. The app still compiles but the warnings tell humans that there are potential issues with the build.